libs/tests/helpers.js

```diff
@@ -58,10 +58,29 @@ async function initializeLibConfig (ownerWallet) {
   }
 }
 
-// Export configured libs instance
-module.exports.audiusInstance = new AudiusLibs(audiusLibsConfig)
-
-// Export libs config for re-use
-module.exports.audiusLibsConfig = audiusLibsConfig
+/**
+ * Advances numberOfBlocks ahead
+ * @param {Web3} web3
+ * @param {number} numberOfBlocks blocks to advance
+ */
+async function advanceBlocks (web3, numberOfBlocks) {
```

Comment:
there's a better way to do this that we use in our contract tests - https://github.com/AudiusProject/audius-protocol/blob/045585c4c69da59d979706ca2e07a5ffad12c914/eth-contracts/test/delegateManager.test.js#L339

can be configured using the provider section here - https://docs.openzeppelin.com/test-helpers/0.5/configuration

Reply:
awesome! i tried this before but totally failed b/c i was against 8545 not 8546 😭 
thanks :) OZ's tools are awesome

Reply:
cool, if it worked let's go ahead and us it in the tests?